apps/yapms/src/lib/components/homepage/HomePageSidebar.svelte

```diff
@@ -1,12 +1,31 @@
 <script lang="ts">
 	import Fa from 'svelte-fa';
 	import { faDiscord, faGithub, faReddit, faTwitter } from '@fortawesome/free-brands-svg-icons';
+
+	import { onMount } from "svelte";
+	import { PocketBaseStore } from '$lib/stores/PocketBase';
+	import SideBarUpdate from './SideBarUpdate.svelte';
+	import type { Record } from 'pocketbase';
+
+	let updates:Record[] = [];
+
+	onMount(async () => {
+		//Request the last 10 update records from pocketbase.
+		$PocketBaseStore.collection('updates').getList(1, 10, {sort: '-created'})
+		.then(data => {
+			updates = data.items
+		}).catch(error => {
+			console.log(`Failed to fetch updates:\n${error}`);
+		});
+	});
```

Comment:
Request:

1) You don't need to use the async/await syntax if you're using callback based promises.

But I do think this should be converted to fully use the async/await syntax with a try/catch.

Avoid using the callback style unless it's really needed.

Reply:
Ah yeah, this is a relic from when I was experimenting with a different implementation. Reworked to a try catch in latest commit.